web/src/components/JsonLdWebSite.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactElement } from 'react'
+import { Helmet } from 'react-helmet'
+import { WebSite, WithContext } from 'schema-dts'
+
+import buildConfig from '../constants/buildConfig'
+
+const JsonLdBreadcrumbs = (): ReactElement => {
+  const config = buildConfig()
+
+  const jsonLd: WithContext<WebSite> = {
+    '@context': 'https://schema.org',
+    '@type': 'WebSite',
+    name: config.hostName,
+    url: window.location.origin,
+    version: __VERSION_NAME__,
```

Comment:
How can for example martin find the version? Just by going to the inspector and looking for it there? Or is there an easier way? Is that okay for him?

Reply:
The Inspecor solution is Martin-Approved ✅